### ISSUE TITLE
用静态变量初始化在调用MMKV.initialize之前初始化是否会产生问题

### ISSUE THREAD

**Issue by OnClickListener2048** [Tencent/MMKV#738]
## Note:
no crash

### The language of MMKV

c++

### The version of MMKV

> e.g. v1.2.7

### The platform of MMKV

android

### The installation of MMKV

Maven

### What's the issue?

> Post the outputs or screenshots for errors.
> 
我这边按照文档在application进行mmkv初始化
  // 设置初始化的根目录
        String dir = getFilesDir().getAbsolutePath() + "/mmkv_pocket";
        String rootDir = MMKV.initialize(dir);
        Log.i("MMKV", "mmkv root: " + rootDir);

但是我在一个静态变量中初始化了mmkv的实例

public static MMKV mmkv = MMKV.mmkvWithID("pocket", MMKV.MULTI_PROCESS_MODE);

我理解的是静态变量初始化会比application更快 所以在mmkv变量赋值的时候 mmkv还没有执行初始化 请问一下凌工这样用是否会导致问题

### What's the log of MMKV when that happened?
no logs

**Comment by lingol** [comment]
First of all, please don't ever call me '凌工', I don't like it.
You should create an MMKV instance only after you call `MMKV.initialize()`.  If you are not sure, you can set a break point to see which one runs first.

**Comment by OnClickListener2048** [comment]
thank you for that

**Comment by 704480904** [comment]
version v1.3.2, page size 4096, arch arm64-v8a

**Comment by lingol** [comment]
> version v1.3.2, page size 4096, arch arm64-v8a Samsung Galaxy Note 10 会出现诡异的闪退问题 说是没有初始化 打印日志 调用方法在init 之后的三秒 只有这一个机型出现了此种问题

You should create a new issue with detail logs from MMKV.
